src/abTestingInfra/ABTestingContext/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
What is `abTestingInfra` supposed to mean, in the filepath? It looks like a lot is included in here. This file — and `Experiment`, `ExperimentContext`, `Variant` — are all React components. Perhaps `src/react/foo.tsx`? Or `src/components/foo.tsx`?

Also — what is the difference between `ABTestingContext` here and `ExperimentContext` below?

Reply:
See comment below on diff between `ABTestingContext ` and `ExperimentContext `. I agreed that we should move all jsx components to the `components` folder.